modules/memfault-firmware-sdk/config/memfault_metrics_heartbeat_extra.def

```diff
@@ -24,6 +24,8 @@ MEMFAULT_METRICS_KEY_DEFINE(Ncs_LteEdrxIntervalMsec, kMemfaultMetricType_Unsigne
 MEMFAULT_METRICS_KEY_DEFINE(Ncs_LteEdrxPtwMsec, kMemfaultMetricType_Unsigned)
 MEMFAULT_METRICS_KEY_DEFINE(Ncs_LteMode, kMemfaultMetricType_Unsigned)
 MEMFAULT_METRICS_KEY_DEFINE(Ncs_LteRsrp, kMemfaultMetricType_Signed)
+MEMFAULT_METRICS_KEY_DEFINE(ncs_lte_data_tx_kilobytes, kMemfaultMetricType_Unsigned)
```

Comment:
Uber nit, can probably drop the `data_` part of the name? Unless you had another metric in mind that would collide otherwise.

Reply:
Good call, completely agree